<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DBus_SerClass</class>
 <widget class="QMainWindow" name="DBus_SerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DBus_Ser</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLineEdit" name="leText"/>
      <widget class="QPushButton" name="btnStart">
       <property name="text">
        <string>开始</string>
       </property>
      </widget>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPlainTextEdit" name="peList"/>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="DBus_Ser.qrc"/>
 </resources>
 <connections/>
</ui>
